CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Add support for `--no-sort` `-U` from [MichaelAug](https://github.com/MichaelAug)
 - Add `--group-directories-first` as an alias for `--group-dirs=first` to improve compatibility with `coreutils/ls`
+- Display MAC and ACL indicators
+- Display MAC labels
```

Comment:
```suggestion
- Display MAC and ACL indicators from [mmatous](https://github.com/mmatous)
```

Reply:
To me + and . are indicators, whatever -Z prints is a context. Changing to 

> Display MAC contexts and MAC and ACL indicators from [mmatous](https://github.com/mmatous)